idf_component.yml

```diff
@@ -3,5 +3,5 @@ url: https://github.com/chmorgan/esp-audio-player
 dependencies:
   ## Required IDF version
   idf:
-    version: ">=4.1.0"
+    version: ">=5.0"
```

Comment:
Shouldn’t we try to keep compatibility with 4.x since 5 hasn’t had a release yet? Is that possible to do?

Reply:
Oh I see 5.0 was released a few days ago! That’s pretty cool. Do you work for espressif btw? Should anyone else from espressif be reviewing changes to this module?

Reply:
> Oh I see 5.0 was released a few days ago! That’s pretty cool. Do you work for espressif btw? Should anyone else from espressif be reviewing changes to this module?

This is a problem, there is currently no automated testing tool for component updates.
Before pushing to GitHub, we'll review together in our Internal GitLab.

Reply:
With this change I presume this module becomes 5.x only? Which is fine but I'd like to understand so I can add a comment to the readme.